### ISSUE TITLE
Esxclude files for dir sources

### ISSUE THREAD

**Issue by koma85** [jordansissel/fpm#1085]
Hello, would be nice exclude files (svn git repositories for example) in dir sources.

**Comment by hatt** [comment]
Have you tried `--exclude`?

**Comment by koma85** [comment]
I swear that i didn't notice the option before.
Sorry for bugging you.

**Comment by jordansissel** [comment]
No worries on not seeing it - fpm has soooo many flags it's easy to get
lost :)

**Comment by hatt** [comment]
Yeah, I wasn't sure if the flag was breaking or you just didn't notice it. The help output is pretty massive, I usually grep for what I want tbh. There is #525 to work on this but it hasn't really gone anywhere in the past year or two...
